### PR TITLE
Hip-351 : Change name of `prng` txn to `utilPrng`

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Related to HIP-351 .
Change name of `prng` transaction to `utilPrng` as per latest discussion, to maintain naming standards for the transaction and transaction body.

### CODE REVIEW

**Review by tinker-michaelj — DISMISSED** [review]
LGTM!

File: services/util_service.proto [hunk 1]
```diff
@@ -34,5 +34,5 @@ service UtilService {
   /**
    * Generates a pseudorandom number.
    */
-  rpc prng (Transaction) returns (TransactionResponse);
+  rpc utilPrng (Transaction) returns (TransactionResponse);
```
Comment (steven-sheehy) [inline services/util_service.proto:37]: This is actually not consistent with other services. Since we're already in the `UtilService` we don't need to specify `util` again. Instead it should just be the `(action)` in `(service name)(action)(TransactionBody)` like we do in crypto_service, etc. So should revert this change

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: services/util_service.proto (hunk 1 above)
Comment (Neeharika-Sompalli) [reply services/util_service.proto:37]: @steven-sheehy  `TokenService` has all transaction names ending with `Token` like `mintToken`, instead of `mint`. Similarly in other services as well.

Comment (steven-sheehy) [reply services/util_service.proto:37]: No service method has the service name first. Some of them have the Hedera entity it's acting upon after the action. This makes sense since the `TokenService.mintToken()` is minting a token, the `CryptoService.createAccount()` is creating an account, etc. The `UtilService.utilPrng()` is not prng'ing a util or util'ing a prng.

Comment (Neeharika-Sompalli) [reply services/util_service.proto]: 😄 Fixed. Thanks !

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by lbaird — APPROVED** [review]
LGTM!
